src/view/com/util/post-ctrls/PostCtrls.tsx

```diff
@@ -363,7 +363,7 @@ let PostCtrls = ({
           threadgateRecord={threadgateRecord}
         />
       </View>
-      {IS_INTERNAL && feedContext && (
```

Comment:
i think we should leave `IS_INTERNAL` in too, it's just another way to get it

Reply:
sure